internal/pkg/template/repository/manager/repository.go

```diff
@@ -185,6 +185,10 @@ func (r *CachedRepository) loadAllTemplates(ctx context.Context) error {
 	wg := &sync.WaitGroup{}
 	errs := errors.NewMultiError()
 	for _, t := range r.repo.Templates() {
+		if t.Deprecated {
+			continue
+		}
+
```

Comment:
Pri starte API servera sa robi pre-load produkcnych templates.
Ale deprecated templates sa nedaju nacitat, zabudol som to tu upravit.

Ako som mergol deprecated templates do `main`, tak API server sa nevedel switchut na novu verziu (ale fungoval dalej).